static/app/components/hovercard.tsx

```diff
@@ -127,10 +127,12 @@ function Hovercard(props: HovercardProps): React.ReactElement {
   // If show is not set, then visibility state is uncontrolled
   const isVisible = props.show === undefined ? visible : props.show;
 
-  const hoverProps = useMemo((): Pick<
-    React.HTMLProps<HTMLDivElement>,
-    'onMouseEnter' | 'onMouseLeave'
-  > => {
+  const hoverProps = useMemo(():
+    | {
+        onMouseEnter: React.MouseEventHandler<HTMLDivElement>;
+        onMouseLeave: React.MouseEventHandler<HTMLDivElement>;
+      }
+    | {} => {
```

Comment:
What's going on this this union? Should these props be optional instead?

Reply:
Yeah, we can also make them optional. I probably went too strict on the type here